### ISSUE TITLE
fatal error: MagickWand/MagickWand.h: No such file or directory

### ISSUE THREAD

**Issue by x0rzkov** [gographics/imagick#243]
Hi guys,

Hope you are all well !

I tried to import the package but I have the following error despite having installed imagemagick on Ubuntu 18.04.

```bash
sudo apt-get install libmagickwand-dev imagemagick
```

```bash
go get -u gopkg.in/gographics/imagick.v3/imagick
go: found gopkg.in/gographics/imagick.v3/imagick in gopkg.in/gographics/imagick.v3 v3.3.0
# gopkg.in/gographics/imagick.v3/imagick
../../../go/pkg/mod/gopkg.in/gographics/imagick.v3@v3.3.0/imagick/affine_matrix.go:8:10: fatal error: MagickWand/MagickWand.h: No such file or directory
 #include <MagickWand/MagickWand.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~
```

Is there something that I am missing ? 

Thanks in advance for any insights on that issue.

Cheers,
X

**Comment by justinfx** [comment]
The default version of ImageMagick available to Ubuntu 18.04 is ImageMagick6. So please use the v2 of this binding library, as per the Readme:

```
go get gopkg.in/gographics/imagick.v2/imagick
```

Otherwise to use v3 you will need to specifically install ImageMagick7

**Comment by x0rzkov** [comment]
Thanks it worked !
